pprof_protobuf/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "pprof-protobuf"
+version = "0.1.0"
+authors = ["Yang Keao <keao.yang@yahoo.com>"]
+edition = "2018"
+
+[features]
+prost-protobuf = ["prost", "prost-derive", "prost-build"]
+rust-protobuf = ["protobuf", "protoc-rust"]
+
+[dependencies]
+prost = { version = "0.6", optional = true }
+prost-derive = { version = "0.6", optional = true }
+protobuf = { version = "2.10", optional = true }
+
+[build-dependencies]
+prost-build = { version = "0.6", optional = true }
+protoc-rust = { version = "2.10.1", optional = true }
```

Comment:
Why not use `protobuf-build`? It should simplify the setup. protoc-rust will introduce a dependency of protoc binary.

Reply:
Great idea!